neumorphic/build.gradle

```diff
@@ -58,7 +59,10 @@ def getArtifactName() {
 
 dependencies {
     api 'androidx.core:core-ktx:1.7.0'
-    api "androidx.compose.ui:ui:$compose_version"
+    api compose.ui
+    api compose.runtime
+    api compose.foundation
+    api compose.material
```

Comment:
Compose runtime , foundation , and material dependency may not be needed here.

Reply:
Merging this commit. if those dependency are redundant. I'll make a separate commit to remove those